third_party/ccid/patches/0003-Add-Sunrex-HP-USB-Business-Slim-Smartcard-CCID-Keybo.patch

```diff
@@ -0,0 +1,100 @@
+From: Ludovic Rousseau <ludovic.rousseau@free.fr>
+Date: Sat, 29 Aug 2020 11:45:10 +0200
+Subject: [PATCH] Add Sunrex HP USB Business Slim Smartcard CCID Keyboard
+
+---
+ ..._Business_Slim_Smartcard_CCID_Keyboard.txt | 58 +++++++++++++++++++
+ readers/supported_readers.txt                 |  5 +-
+ 2 files changed, 62 insertions(+), 1 deletion(-)
+ create mode 100644 readers/Sunrex_HP_USB_Business_Slim_Smartcard_CCID_Keyboard.txt
+
+diff --git a/readers/Sunrex_HP_USB_Business_Slim_Smartcard_CCID_Keyboard.txt b/readers/Sunrex_HP_USB_Business_Slim_Smartcard_CCID_Keyboard.txt
+new file mode 100644
+index 0000000..d5a2e14
+--- /dev/null
++++ b/readers/Sunrex_HP_USB_Business_Slim_Smartcard_CCID_Keyboard.txt
+@@ -0,0 +1,58 @@
++ idVendor: 0x05AF
++  iManufacturer: Sunrex
++ idProduct: 0x605A
++  iProduct: HP USB Business Slim Smartcard CCID Keyboard
++ bcdDevice: 2.11 (firmware release?)
++ bLength: 9
++ bDescriptorType: 4
++ bInterfaceNumber: 1
++ bAlternateSetting: 0
++ bNumEndpoints: 3
++  bulk-IN, bulk-OUT and Interrupt-IN
++ bInterfaceClass: 0x0B [Chip Card Interface Device Class (CCID)]
++ bInterfaceSubClass: 0
++ bInterfaceProtocol: 0
++  bulk transfer, optional interrupt-IN (CCID)
++ iInterface: HP Skylab Smartcard Reader
++ CCID Class Descriptor
++  bLength: 0x36
++  bDescriptorType: 0x21
++  bcdCCID: 1.10
++  bMaxSlotIndex: 0x00
++  bVoltageSupport: 0x07
++   5.0V
++   3.0V
++   1.8V
++  dwProtocols: 0x0000 0x0003
++   T=0
++   T=1
++  dwDefaultClock: 4.800 MHz
++  dwMaximumClock: 12.000 MHz
++  bNumClockSupported: 0 (will use whatever is returned)
++   IFD does not support GET CLOCK FREQUENCIES request: LIBUSB_ERROR_PIPE
++  dwDataRate: 12903 bps
++  dwMaxDataRate: 412903 bps
++  bNumDataRatesSupported: 0 (will use whatever is returned)
++   IFD does not support GET_DATA_RATES request: LIBUSB_ERROR_PIPE
++  dwMaxIFSD: 254
++  dwSynchProtocols: 0x00000000
++  dwMechanical: 0x00000000
++   No special characteristics
++  dwFeatures: 0x000104BA
++   ....02 Automatic parameter configuration based on ATR data
++   ....08 Automatic ICC voltage selection
++   ....10 Automatic ICC clock frequency change according to parameters
++   ....20 Automatic baud rate change according to frequency and Fi, Di params
++   ....80 Automatic PPS made by the CCID
++   ..04.. Automatic IFSD exchange as first exchange (T=1)
++   01.... TPDU level exchange
++  dwMaxCCIDMessageLength: 271 bytes
++  bClassGetResponse: 0xFF
++   echoes the APDU class
++  bClassEnvelope: 0xFF
++   echoes the APDU class
++  wLcdLayout: 0x0000
++  bPINSupport: 0x03
++   PIN Verification supported
++   PIN Modification supported
++  bMaxCCIDBusySlots: 1
+diff --git a/readers/supported_readers.txt b/readers/supported_readers.txt
+index 1fcb100..444ac14 100644
+--- a/readers/supported_readers.txt
++++ b/readers/supported_readers.txt
+@@ -1,6 +1,6 @@
+ #
+ # List of readers supported by the CCID driver
+-# Generated: 2020-08-26
```

Comment:
nit: This is probably irrelevant, but this according to the github review interface this used to be 2020-06-25

Reply:
This is taken from the upstream patch LudovicRousseau/CCID@48680b3 as-is, so let's probably keep it. You're right that in our repo the timestamp is different, but that's practically unavoidable, as long as we start cherry-picking patches from the middle of history.